### ISSUE TITLE
Feature request: Add misp-galaxies to automatic crawlers overview (or add a comment section to display)

### ISSUE THREAD

**Issue by certrik** [ail-project/ail-framework#25]
It would be great to be able to see assigned misp-galaxies in the automatic crawler overview at `/crawlers/auto_crawler` in order to have a better outline of the purpose of tracked pages. Alternatively also some kind of comment ability to add to automatic crawlers would be great. This information also could be displayed in the overview.

**Comment by adulau** [comment]
Good idea, we will have a look to add this in the next versions. Thank you.
